ofrak_core/test_ofrak/unit/resource_view/test_view.py

```diff
@@ -257,13 +257,16 @@ async def test_resource_view_updates(ofrak_context: OFRAKContext):
     await instr_view.resource.run(
         InstructionModifier, InstructionModifierConfig("add", "r4, r5", InstructionSetMode.NONE)
     )
+
+    assert instr_view.mnemonic == "add"
+    assert instr_view.operands == "r4, r5"
+
     instr_r.add_view(
-        Instruction(0x100, 0x4, "add r4, r5", "add", "r4, r5", InstructionSetMode.NONE)
+        Instruction(0x100, 0x4, "sub r4, r5", "sub", "r4, r5", InstructionSetMode.NONE)
     )
     await instr_r.save()
 
-    assert instr_view.mnemonic == "add"
-    assert instr_view.operands == "r4, r5"
+    assert instr_view.mnemonic == "sub"
```

Comment:
This test confused me. Part of the reason is that it is testing two different behaviors (and the original test didn't make it clear what it was testing).

I'd appreciate it if it was split into tow different tests:
```
# test that InstructionModifier changes propogate to resource view.
await instr_view.resource.run(
        InstructionModifier, InstructionModifierConfig("add", "r4, r5", InstructionSetMode.NONE)
    )

    assert instr_view.mnemonic == "add"
    assert instr_view.operands == "r4, r5"
```
and
```
    instr_r.add_view(
        Instruction(0x100, 0x4, "add r4, r5", "add", "r4, r5", InstructionSetMode.NONE)
    )
    await instr_r.save()

assert instr_view.mnemonic == "add"
    assert instr_view.operands == "r4, r5"
```